src/katsdpcontroller/generator.py

```diff
@@ -1513,10 +1490,128 @@ def get_baseline_index(a1, a2):
         )
         xbgpu.static_gauges["xbgpu_expected_engines"] = 1.0
 
-    _add_task_sensors(g, streams, task_names)
     return xbgpu_group
 
 
+def _make_vgpu(
+    g: networkx.MultiDiGraph,
+    configuration: Configuration,
+    stream: product_config.GpucbfTiedArrayResampledVoltageStream,
+) -> scheduler.LogicalNode:
+    n_engines = 1
+    n_substreams = 1
+    vgpu = ProductLogicalTask(f"vgpu.{stream.name}", streams=[stream])
+    vgpu.subsystem = "cbf"
+    vgpu.image = "katgpucbf"
```

Comment:
```suggestion
    vgpu.image = "katgpucbf"
    # vgpu doesn't use katsdpservices for configuration, or telstate
    vgpu.katsdpservices_config = False
    vgpu.pass_telstate = False
```
Otherwise a telstate will be started, and that's what's causing you to need to add a sleep to the reuse_name unit test.